### ISSUE TITLE
Getting a bunch of "deprecated documentation syntax" when running the bot

### ISSUE THREAD

**Issue by emilsoman** [code-mancers/hubot-pr-fu#18]
Here's the output that I got : 

```
[Mon Sep 28 2015 14:44:36 GMT+0530 (IST)] INFO Connecting...
[Mon Sep 28 2015 14:44:39 GMT+0530 (IST)] INFO Logged in as eva of codemancers, but not yet connected
[Mon Sep 28 2015 14:44:41 GMT+0530 (IST)] INFO Slack client now connected
[Mon Sep 28 2015 14:44:41 GMT+0530 (IST)] INFO /Users/emil/Testing/hubot-slack/node_modules/hubot-pr-status/scripts/hubot-pr-status.coffee is using deprecated documentation syntax
[Mon Sep 28 2015 14:44:41 GMT+0530 (IST)] INFO /Users/emil/Testing/hubot-slack/node_modules/hubot-pr-status/scripts/post_merge_hook.coffee is using deprecated documentation syntax
[Mon Sep 28 2015 14:44:41 GMT+0530 (IST)] INFO /Users/emil/Testing/hubot-slack/node_modules/hubot-pr-status/scripts/status_all.coffee is using deprecated documentation syntax
[Mon Sep 28 2015 14:44:41 GMT+0530 (IST)] INFO /Users/emil/Testing/hubot-slack/node_modules/hubot-pr-status/scripts/status_conflicts.coffee is using deprecated documentation syntax
[Mon Sep 28 2015 14:44:41 GMT+0530 (IST)] INFO /Users/emil/Testing/hubot-slack/node_modules/hubot-pr-status/scripts/status_user.coffee is using deprecated documentation syntax
```

**Comment by emilsoman** [comment]
If we document the commands as per https://github.com/github/hubot/blob/master/docs/scripting.md#documenting-scripts , these will show up when a user types "@bot help"

**Comment by kgrz** [comment]
Will do this tonight

**Comment by kgrz** [comment]
Fixed by #27
